### ISSUE TITLE
Crashes on Ubuntu 18.04 due to different glibc version

### ISSUE THREAD

**Issue by MortronMeymo** [esrlabs/chipmunk#1042]
**Prerequisites**

affected version: Latest (can't really check due to crash)

**Describe the issue**
Latest version requires glibc 2.29 however Ubuntu 18.04 only comes with glibc 2.27.

```
ldd --version
ldd (Ubuntu GLIBC 2.27-3ubuntu1.4) 2.27
```

**Steps to Reproduce**

Try running chipmunk on Ubuntu 18.04

**Environment**

Ubuntu 18.04

**Failure Logs**

```
[JS]: 1612778717717: [BAD] UncaughtException: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /.../resources/app/node_modules/indexer-neon/native/index.node)```

**Comment by marcmo** [comment]
Thanks for the report, we will take a look at it

**Comment by marcmo** [comment]
should be solved with the next release snapshot probably tomorrow

**Comment by marcmo** [comment]
solved with 2.13.*
